libsolidity/interface/SourceReferenceFormatter.cpp

```diff
@@ -31,8 +31,11 @@ namespace dev
 namespace solidity
 {
 
+bool g_hasOutputed = false;
```

Comment:
I'm not a native speaker, but I think it should be just `g_hasOutput`.

Reply:
sure, thx, will change it soon